internal/runner/executer.go

```diff
@@ -23,21 +23,28 @@ func (r *Runner) ExecuteCrawling() error {
 
 	wg := sizedwaitgroup.New(r.options.Parallelism)
 	for _, input := range inputs {
+		if input == "" {
+			gologger.Warning().Msgf("Skipping empty input")
+			continue
+		}
 		if !r.networkpolicy.Validate(input) {
 			gologger.Info().Msgf("Skipping excluded host %s", input)
 			continue
 		}
 		wg.Add()
 		input = addSchemeIfNotExists(input)
+		if r.crawler == nil {
```

Comment:
We can guard against at the beginning of this function. That way, nothing is done if crawler is nil.